src/applications/simple-forms/21-0845/pages/authorizerType.js

```diff
@@ -3,24 +3,27 @@ import React from 'react';
 import { AUTHORIZER_TYPE_ITEMS } from '../definitions/constants';
 import { getEnumsFromConstants, getLabelsFromConstants } from '../utils';
 
+const labelString = 'Who is submitting this authorization?';
+
 /** @type {PageSchema} */
 export default {
   uiSchema: {
     authorizerType: {
       'ui:title': (
         <>
-          <span className="custom-header authorizer-type vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--h3">
-            Who is submitting this authorization?{' '}
-            <span className="custom-required-span hide-on-review-page">
-              (*Required)
-            </span>
-          </span>
-          <p className="custom-description hide-on-review-page">
-            Select the description that fits you.
-          </p>
+          <h2 className="vads-u-font-size--h3">{labelString}</h2>
+          <p>Select the description that fits you.</p>
```

Comment:
just jotting notes from our discussion - 
Problem 1:
- ui:description does not get read by screen readers. This is a problem for every form. not a concern of this form in particular. However it is a launch blocker for this form, and that's why this solution is being done like this.
- TODO: We want a solution that every form can use, we want to use ui:title and ui:description separately, most likely still.

Problem 2:
- using a h2 will cause this header to also show up on the review page. this is a problem for every form is it not? a temporary solution for this form is to add a custom review field override.
- TODO: We want a solution for every form. A solution might be to add a headerLabelText: '2' and a headerClassNames uiOption. Just an idea.